frontend/tests/integration/components/transportation/study-area-map-test.js

```diff
@@ -80,4 +80,35 @@ module('Integration | Component | transportation/study-area-map', function(hooks
 
     assert.ok(find("[data-test-popup='census-tract']"));
   });
+
+  test('it selected features on click', async function(assert) {
+    // If a project exists with a transportation analysis
+    const project = server.create('project');
+    this.model = await this.owner.lookup('service:store')
+      .findRecord('project', project.id, { include: 'transportation-analysis'});
+
+    let events = {};
+    const geoid = 'test';
+    this.map = {
+      ...DEFAULT_MAPBOX_GL_INSTANCE,
+      queryRenderedFeatures() {
+        return [{
+          type: 'Feature',
+          properties: {
+            geoid: geoid,
+          },
+        }]
+      },
+      on: (event, action) => {
+        events[event] = action;
+      }
+    };
+
+    await render(hbs`{{transportation/study-area-map analysis=model.transportationAnalysis}}`);
+    events['click']({point: 'test'});
```

Comment:
The shape of this should be: 

```
{ point: { x: 0, y: 0 } } 
```

Reply:
technically yes, but it doesn't matter and nothing that I'm testing here requires that shape :)